content/blog/bare-metal-provisioning-on-hpe-greenlake-using-terraform.md

```diff
@@ -0,0 +1,359 @@
+   \
+   This will be exported in the environment variable **HPE_METAL_PROJECT_ID** in the later section.
+
+![Compute Group ID](/img/compute_group_id.png "Compute Group ID")
+
+ ﻿ 
+
+## Install Terraform
+
+Next, get your system ready to run Terraform. In case this has not been done yet:
+
+1. Download and install Terraform, version v0.13 or later.
+   For more information, see [https://learn.hashicorp.com/tutorials/terraform/install-cli](<1. https://learn.hashicorp.com/tutorials/terraform/install-cli>). 
+2. Verify the installation with **terraform -help.** 
+
+   At this point, you are ready to start building your infrastructure description file. 
+
+# Deploy Compute Instance
+
+## Select Terraform provider with bare metal service configurations
+
+1. #### Export the following environment variables on your setup.
+
+   E﻿xport the Tenant ID:
+
+   ```shell
+   export HPEGL_TENANT_ID="<Tenant ID>
+   ```
+
+   Export the API Client credentials that you obtained when you created an API Client within HPE GreenLake Central:
+
+   ```shell
+   export HPEGL_USER_ID="<API Client ID>"
+   export HPEGL_USER_SECRET="<API Client Secret>"
+   export HPEGL_IAM_SERVICE_URL="<Issuer URL>"
+   ```
+
+   E﻿xport the Compute Group ID:
+
+   ```
+   # compute group/project ID
+   export HPEGL_METAL_PROJECT_ID="<Compute Group ID>"
+   ```
+
+    ﻿Export bare metal service REST URL:
+
+   ```
+   # Production Environment:  https://client.greenlake.hpe.com/api/metal
+   # Integration Environment: https://client.greenlake.hpe-gl-intg.com/api/metal
+   # local development: http://localhost:3002
+
+   export HPEGL_METAL_REST_URL="<Metal Service REST Base URL>"
+   ```
+2. #### Configure the Terraform provider.
+
+   Create an empty folder and put a file in it called main.tf with the following contents:
+
+   **main.tf:**
+
+   ```hcl
+   terraform {
+     required_providers {
+       hpegl = {
+         source  = "HPE/hpegl"
+         version = ">= 0.3.12"
+       }
+     }
+   }
+
+   provider "hpegl" {    
+     # metal block for configuring bare metal resources.
+     metal {   
+     } 
+   }
+   ```
+
+   T﻿his tells Terraform that you are going to be using HPE/hpegl as your provider and you are using bare metal service.
+
+## Write resource configuration for Compute Instance creation
+
+To deploy compute instance, you need to use the **hpegl_metal_host** terraform resource.
+
+***Note**:*
+
+* *compute instance is AKA host.* 
+* *compute instance type is AKA machine size.*
+
+The **hpegl_metal_host** resource supports many different arguments, but these are the required ones:
+
+* networks – List of networks. The list must always include any networks marked as '**required**' for that location. 
+* machine_size – Compute Instance type.
+* ssh – List of SSH keys to push to the host.
+* location – The data center location of where the compute instance will be provisioned.
+* Image – OS image in the form flavor@version.
```

Comment:
This particular bullet has increased in font size - you'll want to double check the formatting. I don't see anything obvious that is doing this... please check with Denis Choukroun to see if he can advise.